src/lib/utils/translations.ts

```diff
@@ -12,12 +12,17 @@ export const languages: Languages = i18nConfig.reduce((result, config) => {
 export const isLangRightToLeft = (lang: Lang): boolean => {
   const langConfig = i18nConfig.filter((language) => language.code === lang)
 
-  if (!langConfig.length)
-    throw new Error("Language code not found in isLangRightToLeft")
+  if (!langConfig.length) {
+    return false
```

Comment:
cc @wackerow I'm removing this since now we have a case where a locale doesn't exist and this will make the builds to fail. I guess we could add another condition to check these faked locales but didn't want to add more logic to this function tbh. LMK what you think and if you prefer to keep the error.

Reply:
Sounds good... realizing we have a handful of locales that are live but not yet included in the `Lang` type ('am', 'be', 'bs', 'hy-am', 'kn', 'ne-np', 'fil', 'tk')... we could probably clean that up separately to enforce this through typing; looks good for this PR though